Model/Carrier/Parcelpro.php

```diff
@@ -75,7 +76,7 @@ protected function _rateresult($key, $value)
 
         $matches = explode('_', $key);
         if ($matches[0] === 'dhl') {
-            $rate->setCarrierTitle($this->getConfigData('dhl_title'));
+            $rate->setCarrierTitle($this->getConfigData('dhl_title') . "line79");
```

Comment:
This debug code should be removed.
```suggestion
            $rate->setCarrierTitle($this->getConfigData('dhl_title'));
```